components/About/Members.js

```diff
@@ -0,0 +1,251 @@
+import Image from "next/image"
+
+const Members = () => {
+  const members = [
+    { name: "Executive Committee",
+      members: [
+        {
+          name: "Mr. Salman Bin Kashif",
+          position: "Chairperson",
+          university: "Clemson University",
+          place: "Clemson, USA",
+          imageUrl: "/members/Salman Bin Kashif.jpg"
+        },
+        {
+          name: "Mr. Syed Ali Rizvi",
+          place: "Boston, Massachusetts, USA",
+          imageUrl: "/members/Syed Ali Rizvi.jpg"
+        },
+        {
+          name: "Dr. Saif Sheikh",
+          place: "Chicago, Illinois, USA",
+          imageUrl: "/members/Saif Sheikh.jpg"
+        },
+        {
+          name: "Dr. Rehan Baqri",
+          place: "Boston, Massachusetts, USA",
+          imageUrl: "/members/Rehan Baqri.jpg"
+        },
+        {
+          name: "Dr. Shaida Andrabi",
+          place: "Kashmir, India",
+          imageUrl: "/members/Shaida Andrabi.jpg"
+        },
+        {
+          name: "Dr. Mohsin Khan",
+          place: "Washington DC, USA",
+          imageUrl: "/members/Mohsin Khan.jpg"
+        },
+        {
+          name: "Mr. Ali Muzaffar",
+          place: "Irvine, CA, USA",
+          imageUrl: "/members/Ali Muzaffar.jpg"
+        },
+        {
+          name: "Mr. Wasikul Islam",
+          place: "Stillwater, Oklahoma, USA",
+          imageUrl: "/members/Wasikul Islam.jpg"
+        }
+      ]
+    },
+    { name: "Mentorship and Applications",
+      members: [
+        {
+          name: "Mr. Shujaut H. Bader",
+          position: "Team Lead",
+          scholar: "SSGSA Scholar 2014-15",
+          university: "Iowa State University",
+          place: "Ames, USA",
+          imageUrl: "/members/Shujaut H. Bader.jpg"
+        },
+        {
+          name: "Mr. Shiva Agarwal",
+          scholar: "SSGSA Scholar 2018-19",
+          university: "Western Michigan University",
+          imageUrl: "/members/Shiva Agarwal.jpeg"
+        },
+        {
+          name: "Mr. Sk Abdus Sayeed",
+          scholar: "ISRA Scholar 2015",
+          university: "Cleveland State University",
+          place: "Cleveland, OH, USA",
+          imageUrl: "/members/Sk Abdus Sayeed.jpg"
+        },
+        {
+          name: "Mr. Sajad Sheikh",
+          scholar: "SSGSA Scholar 2016-17",
+          university: "Kent State University",
+          place: "OH, USA",
+          imageUrl: "/members/Sajad Sheikh.jpeg"
+        },
+        {
+          name: "Mr. Md Ful Hossain Sk",
+          scholar: "SSGSA Scholar 2018-19",
+          university: "University of Kansas Lawrence",
+          place: "Kansas USA",
+          imageUrl: "/members/Ful Hossain.jpeg"
```

Comment:
Just a small change require @himanshuvarandani. Remove all the whitespace from the image name. Those images won't load on the website. Use `-` instead of space.